### PR TITLE
MM-8374 - Updating docs styling

### PR DESCRIPTION
**Pull request by asaadmahmood**
Working on this for this ticket: https://mattermost.atlassian.net/browse/MM-8374

I've updated the styling of the blog list and single blog pages, along with some other minor global changes.

The subscribe box can be a separate ticket as I would need a dev to plug in the markup for me first and then I can go and style it.

### CODE REVIEW

**Comment by asaadmahmood** [comment]
@jasonblais @esethna Whoever reviews docs now.

**Comment by jasonblais** [comment]
Thanks @asaadmahmood! Looking great

1 - Looks like the header isn't vertically centered

image

2 - The formatting breaks on narrow view (when title goes into the date area)

image

3 - For an individual blog post, the word count, categories and tags appear at the bottom. Not sure if that's most obvious?

image

File: site/static/css/styles.css [hunk 1]
```diff
@@ -2371,11 +2375,20 @@ h4,
 h5,
 h6 {
   font-weight: 400;
-  margin-bottom: 1em; }
+  margin-bottom: 1em;
+  margin-top: 1.8em; }
```
Comment (lieut-data) [inline site/static/css/styles.css]: nit: locate bracket on next line?
```suggestion
  margin-top: 1.8em;
}
```

File: site/static/css/styles.css [hunk 2]
```diff
@@ -2371,11 +2375,20 @@ h4,
...
 
 .btn {
   cursor: pointer;
-  padding: .8em 2em; }
+  padding: .6em 1.3em; }
```
Comment (lieut-data) [inline site/static/css/styles.css]: Same as above:
```suggestion
  padding: .6em 1.3em; 
}
```

**Pull request merged**